### PR TITLE
added a new model for "Hey GlaDOS"

### PR DESCRIPTION
**Pull request by JohnTheNerd**
it's horribly overfit with over a million steps on an RTX 3090 (~24 hours of total training time), with 30k samples before augmenting. the loss stayed at 0.000 long enough for you to read [the grokking paper](https://arxiv.org/abs/2201.02177) hundreds of times.

despite the rather discouraging two sentences I just wrote, the model actually works shockingly well. never had any false positives thus far and the wake word seems to work even in poor conditions, on the atom echo no less. tested by blasting Thunderstruck by AC/DC on my speaker while shouting out "hey glados" which definitely got me some amusing messages from my roommates.

used the Jupyter notebook given in the microwakeword repository with some edits that I'll commit when I get a chance.

target words (created 2000 samples for each, using the Piper sample generator):

- hey_glahdohs
- hey_glahdos
- hey_glahdows
- hey_glahdohss
- hey_glados
- hey_glaados
- hey_gladdos
- hey_gladoughs
- hey-glahdohs
- hey-glahdos
- hey-glahdows
- hey-glados
- hey-glaados
- hey-gladdos
- hey-gladoughs

noise-scales and noise-scale-ws was set to 0.5, min-phoneme-count was set to 10, positive class weight was 2, negative class weight was 100, and the minimization metric was set to ambient_false_positives_per_hour. the target for minimization was set to 0.01 but according to the validation we got something much lower in practice.

### CODE REVIEW

**Comment by JohnTheNerd** [comment]
after running the model for a few weeks, I have noticed false negatives are still very rare but false positives seem to happen more often than I'd like. I will update and re-open after I am more confident in its accuracy

**Comment by drkhsh** [comment]
@JohnTheNerd Thanks for the model 💯
Any updates in the meanwhile?
